nunaliit2-couch-application/src/main/atlas_couchapp/_attachments/tools/js/metadata.js

```diff
@@ -0,0 +1,395 @@
+;(function ($, $n2) {
+    "use strict";
+
+    // Localization
+    var _loc = function (str, args) {
+        return $n2.loc(str, 'nunaliit2-couch', args);
+    };
+
+    var DH = 'metadata.js';
+
+    var defaultMetadata = JSON.parse('{\n' +
+        '        "license": "",\n' +
+        '        "creator": {\n' +
+        '          "contactPoint": {\n' +
+        '            "@type": "ContactPoint",\n' +
+        '            "contactType": "",\n' +
+        '            "telephone": "",\n' +
+        '            "email": ""\n' +
+        '          },\n' +
+        '          "@type": "Organization",\n' +
+        '          "name": "",\n' +
+        '          "url": ""\n' +
+        '        },\n' +
+        '        "dateCreated": "",\n' +
+        '        "keywords": [],\n' +
+        '        "temporalCoverage": "",\n' +
+        '        "@type": "Dataset",\n' +
+        '        "name": "",\n' +
+        '        "description": "",\n' +
+        '        "spatialCoverage": {\n' +
+        '          "geo": {\n' +
+        '            "@type": "GeoShape",\n' +
+        '            "box": ""\n' +
+        '          },\n' +
+        '          "@type": "Place"\n' +
+        '        },\n' +
+        '        "@context": {\n' +
+        '          "@vocab": "http://schema.org"\n' +
+        '        },\n' +
+        '        "url": ""\n' +
+        '      }');
+
+    var config = null;
+    var authService = null;
+    var dispatcher = null;
+    var atlasDb = null;
+    var atlasDesign = null;
+    var schemaRepository = null;
+    var showService = null;
+    var couchEditor = null;
+    var $metadataAppDiv = null;
+
+    function getDocumentDiv() {
+        var $docDiv = $('#metadataAppDocument');
```

Comment:
@ahayes prefers if Nunaliit doesn't use html element classes/ids which are in camelCase. For the most part we tend to use camelCase in our js and usually snake_case for html and css. Note: You will see some exceptions to this in our older code e.g. `n2_documentDisplay`, but typically we tend to stick to snake_case now.
```suggestion
        var $docDiv = $('#metadata_app_document');
```

Reply:
Fixed.  You'll have to add some of these rules to our soon-to-be "Style Guide".